### PR TITLE
Add Github actions 

### PR DESCRIPTION
**Pull request by epourail**
| Q | A |
| - | - |
| Bug Fix ? | no |
| New Feature ? | yes |
| Issue | #21 |
 
Checklist:

- [x] Bug fix are covered by unit tests 
- [x] New feature are covered by unit,api and acceptance tests
- [x] All tests passed
- [x] Code style is respected
- [x] Commit message explains **why** the change is made (see https://github.com/erlang/otp/wiki/Writing-good-commit-messages)
- [x] CHANGELOG.md contains a short summary of the change
- [x] Documentation is updated as necessary

### Why this change is needed?
Define the github actions to build the docker images and run the tests

### CODE REVIEW

**Comment by Ealenn** [comment]
It's ugly, even if you don't deploy any artifacts the name and description is required https://getcomposer.org/doc/04-schema.md#name

File: .github/workflows/docker-run-tests.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: Run the Tests (unit, api)
```
Comment (Ealenn) [inline .github/workflows/docker-run-tests.yml]: Between us, the name of the file isn’t representative 🙃 `run-tests.yaml` ? `php-run-tests.yaml` ?

File: .github/workflows/docker-run-tests.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        
+      - name: Validate composer.json and composer.lock
+        run: composer validate
```
Comment (Ealenn) [inline .github/workflows/docker-run-tests.yml]: --no-check-publish https://getcomposer.org/doc/03-cli.md#validate 😋

Comment (Ealenn) [reply .github/workflows/docker-run-tests.yml]: It's ugly, even if you don't deploy any artifacts the name and description is required https://getcomposer.org/doc/04-schema.md#name either you modify your test, or you add this information

File: .github/workflows/docker-run-tests.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+on:
+  pull_request:
+    branches:
+      - master
+      - feature/*
+      - fix/*
+
+jobs:
+  build:
```
Comment (Ealenn) [inline .github/workflows/docker-run-tests.yml]: You can name this job `tests`

File: .github/workflows/docker-run-tests.yml (hunk 2 above)
Comment (epourail) [reply .github/workflows/docker-run-tests.yml]: I removed this step as it was defined in the "php" github actions template.

**Pull request merged**